components/ingest-service/backend/elastic/elastic.go

```diff
@@ -42,6 +42,24 @@ func (es *Backend) Initializing() bool {
 	return !es.initialized
 }
 
+func (es *Backend) DeleteProjectTag(ctx context.Context, projectTagToBeDelete string) ([]string, error) {
+	log.Debugf("starting project delete project %q", projectTagToBeDelete)
+
+	esNodeJobID, err := es.DeleteNodeProjectTag(ctx, projectTagToBeDelete)
+	if err != nil {
+		return []string{}, errors.Wrap(err, "Failed to start Elasticsearch Node project tags delete")
+	}
+
+	esActionJobID, err := es.DeleteActionProjectTag(ctx, projectTagToBeDelete)
+	if err != nil {
+		return []string{}, errors.Wrap(err, "Failed to start Elasticsearch Action project tags delete")
+	}
+
+	log.Debugf("Started Project delete with node job ID: %q action job ID %q", esNodeJobID, esActionJobID)
```

Comment:
When I read this, it looks like we could have successfully deleted the project tag from Elasticsearch nodes but failed to do so from Elasticsearch Actions. Does that leave things in a confusing state? Do they need to both happen, or should we at least have an indication that one succeeded? (also i guess none of this is suggestive of success, just starting)

Reply:
If the starting of one of the jobs fails most likely the other will fail. This would normally happen when Elasticsearch is in a bad state. This does leave things in a confusing state, but if we can not start the delete job there is not much else we can do. The only way this confusing state will be cleaned up is by running an apply-rules request.